lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -26,6 +26,16 @@ using namespace mlir;
 
 namespace circt {
 
+//===----------------------------------------------------------------------===//
+// Utility types
+//===----------------------------------------------------------------------===//
+
+/// A mapping is maintained between a function operation and its corresponding
+/// Calyx component. This facilitates translation when function and Calyx name
+/// are not identical, such as when the top-level function is renamed to
+/// 'main' to comply with Calyx conventions.
```

Comment:
The latter half of the comment needs to be updated, right?

Reply:
Correct, thank you for spotting that :+1:.